### DISCUSSION TITLE
Port 8000 not published

### DISCUSSION THREAD

**Discussion by hexclann** [plausible/analytics#1850] [Self-Hosted Support]
Hello, I followed the official doc and deployed the containers but the port is not mapped. I tried changing the port in `docker-compose.yml` but no luck, what am I doing wrong?

![chrome_s4cIcq3VEQ](https://user-images.githubusercontent.com/59732888/164896095-f0a192fd-836b-48d2-8a55-7cf5a37be30f.png)

**Comment by hexclann** [comment]
Update: I'm trying this on ARM architecture so all the containers were stuck in a restarting loop. However I tried to run https://github.com/plausible/hosting/pull/45 by @paskal but `bytemark/smtp` and `plausible/analytics:latest` containers were stuck in a restart loop

Logs:

```
....
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
standard_init_linux.go:228: exec user process caused: exec format error
...
```

**Reply by paskal** [reply]
Plausible itself currently wouldn't work. You need to do following for it to work on the same machine you're running Plausible:

```shell
git clone https://github.com/plausible/analytics.git
cd analytics.git
docker build --pull -t plausible/analytics:latest  .
```

After that, you'll have a properly built image for ARM. After starting the container again, it will work properly.
